### PR TITLE
docs: correct grammatical errors in 1.installation.md

### PR DESCRIPTION
**Pull request by idorenyinudoh**
Include commas and more descriptive texts where necessary.

### CODE REVIEW

File: content/en/docs/1.get-started/1.installation.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ You can play with Nuxt online directly on CodeSandbox or StackBlitz:
 
 ## Using create-nuxt-app
 
-To get started quickly you can use the [create-nuxt-app](https://github.com/nuxt/create-nuxt-app).
+To get started quickly, you can use the [create-nuxt-app](https://github.com/nuxt/create-nuxt-app).
```
Comment (danielroe) [inline content/en/docs/1.get-started/1.installation.md:32]:
```suggestion
To get started quickly, you can use [create-nuxt-app](https://github.com/nuxt/create-nuxt-app).
```

File: content/en/docs/1.get-started/1.installation.md [hunk 2]
```diff
@@ -29,9 +29,9 @@ You can play with Nuxt online directly on CodeSandbox or StackBlitz:
...
 
-Make sure you have npx installed (npx is shipped by default since npm 5.2.0) or npm v6.1 or yarn.
+Make sure you have npx (npx is shipped by default since npm 5.2.0), npm (v6.1 or later) or yarn installed.
```
Comment (danielroe) [inline content/en/docs/1.get-started/1.installation.md:34]:
```suggestion
Make sure you have installed yarn, npx (included by default with npm v5.2+) or npm (v6.1+).
```

File: content/en/docs/1.get-started/1.installation.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ npm run dev
 The application is now running on [http://localhost:3000](http://localhost:3000). Well done!
 
 ::alert{type="info"}
-Another way to get started with Nuxt.js is to use [CodeSandbox](https://template.nuxtjs.org) which is a great way for quickly playing around with Nuxt.js and/or sharing your code with other people.
+Another way to get started with Nuxt.js is to use [CodeSandbox](https://template.nuxtjs.org), which is a great way for quickly playing around with Nuxt.js and/or sharing your code with other people.
```
Comment (danielroe) [inline content/en/docs/1.get-started/1.installation.md:66]:
```suggestion
Another way to get started with Nuxt is to use [CodeSandbox](https://template.nuxtjs.org), which is a great way for quickly playing around with Nuxt and/or sharing your code with other people.
```

File: content/en/docs/1.get-started/1.installation.md [hunk 4]
```diff
@@ -122,7 +122,7 @@ npm install nuxt
 ```
 ::
 
-This command will add `nuxt` as a dependency to your project and add it to your `package.json`. The `node_modules` directory will also be created which is where all your installed packages and dependencies are stored.
+This command will add `nuxt` as a dependency to your project and include it in your `package.json`. The `node_modules` directory will also be created which is where all your installed packages and dependencies are stored.
```
Comment (danielroe) [inline content/en/docs/1.get-started/1.installation.md:125]: I think the original was clearer here.
```suggestion
This command will add `nuxt` as a dependency to your project and add it to your `package.json`. The `node_modules` directory will also be created which is where all your installed packages and dependencies are stored.
```

**Review by danielroe — COMMENTED** [review]
Thanks ❤️

**Comment by idorenyinudoh** [comment]
@danielroe You're welcome! I think you forgot to merge the pr.

File: content/en/docs/1.get-started/1.installation.md (hunk 4 above)
Comment (idorenyinudoh) [reply content/en/docs/1.get-started/1.installation.md:125]: I think "include" is the more appropriate word but okay.

**Comment by idorenyinudoh** [comment]
@danielroe I've included your suggestions. Thank you!

**Comment by idorenyinudoh** [comment]
Closed this pr due to conflicts but resolved them and opened [another one](https://github.com/nuxt/nuxtjs.org/pull/1740)

**Pull request closed without merging**